Dockerfile-crosseth

```diff
@@ -49,26 +49,15 @@ RUN apt-get install -y \
 RUN apt-get install -y g++-arm-linux-gnueabi g++-arm-linux-gnueabihf
 
 # Switch to a normal user account.
-RUN useradd -ms /bin/bash xcompiler
-USER xcompiler
+RUN useradd -ms /bin/bash crosseth
+USER crosseth
```

Comment:
using a different user for the "eth" part now. I also use those 2 strings ("xcompiler" and "crosseth") throughout to distinguish which of the two docker file/wrappers we are using